src/api.ts

```diff
@@ -12,7 +11,7 @@ import {
 import { getParameter } from "./apis/parameters";
 import { getLogger } from "./logger/logger";
 
-const basToolkitAPI: Omit<BasToolkit, "workspaceAPI"> = {
+const basToolkitAPI = {
```

Comment:
Why did you remove the type signature?
We are losing type checking on the object literal here...

Reply:
Done